src/markdown-pages/build-apps/add-time-picker-guide.mdx

```diff
@@ -21,28 +21,48 @@ We also have a 12 minute video that covers the steps below.
 
 ## Before you begin
 
-These steps require that you have the following:
+To develop projects, you need our New Relic One CLI (command line interface). If you haven't already installed it, do the following:
 
-- A GitHub account
-- New Relic developer account
-- API Key (the same key you set up in [Quick start](https://one.newrelic.com/launcher/developer-center.launcher?pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==))
-- The New Relic One [CLI](https://cli.nr-ext.net/installer.pkg) 
+* Install [Node.js](https://nodejs.org/en/download/).
+* Complete steps 1–4 of our [CLI wizard](https://one.newrelic.com/launcher/developer-center.launcher?pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==), and be sure to make a copy of your account ID from step 1 because you’ll need it later.
+
+<Important>
+
+  If you already installed the New Relic One CLI, but you can't remember your account ID, you can re-run the wizard just to get your account ID.
+
+</Important>
+
+For additional details, see [Set up your development environment](https://developer.newrelic.com/build-apps/set-up-dev-env).
 
 ## Prepare the time picker sample code
 
-To get started, complete these steps to update the application UUID and run the sample application locally:
+To get started, complete these steps to update the [application UUID (unique ID)](https://developer.newrelic.com/build-tools/new-relic-one-applications/guide-to-authentication--data-access--and-permissions) and run the sample application locally:
```

Comment:
That page no longer exists. please link to the CLI page here: (/explore-docs/nr1-cli)

Reply:
I'm adding.